specifyweb/specify/datamodel.py

```diff
@@ -8161,6 +8161,38 @@
         ],
         fieldAliases=[
 
+        ]
+    ),
+    Table( # CollectionObjectType
+        sp7_only=True,
+        django_app='specify',
+        classname='edu.ku.brc.specify.datamodel.CollectionObjectType',
+        table='collectionobjecttype',
+        tableId=1013,
+        idColumn='CollectionObjectTypeID',
+        idFieldName='collectionObjectTypeId',
+        idField=IdField(name='collectionObjectTypeId', column='collectionObjectTypeID', type='java.lang.Integer'),
+        fields=[
+            Field(name='name', column='Name', indexed=False, unique=False, required=True, type='java.lang.String', length=255),
+            Field(name='isloanable', column='IsLoanable', indexed=False, unique=False, required=True, type='java.lang.Boolean'),
+            Field(name='version', column='Version', indexed=False, unique=False, required=False, type='java.lang.Integer'),
+            Field(name='timestampcreated', column='TimestampCreated', indexed=False, unique=False, required=True, type='java.sql.Timestamp'),
+            Field(name='timestampmodified', column='TimestampModified', indexed=False, unique=False, required=False, type='java.sql.Timestamp'),
+            Field(name='text1', column='Text1', indexed=False, unique=False, required=False, type='java.lang.String', length=255),
+            Field(name='text2', column='Text2', indexed=False, unique=False, required=False, type='java.lang.String', length=255),
+            Field(name='text3', column='Text3', indexed=False, unique=False, required=False, type='java.lang.String', length=255)
+        ],
+        indexes=[
+
+        ],
+        relationships=[
+            Relationship(name='collection', type='many-to-one', required=False, relatedModelName='Collection', column='Collection_ID'),
+            Relationship(name='taxontreedef', type='many-to-one', required=False, relatedModelName='TaxonTreeDef', column='TaxonTreeDef_ID'),
```

Comment:
Good question, I talked to Grant, we are just doing taxon trees.  In the future we might add Geography trees, but it will likely be above discipline in scope.  So, I'm going to leave it as is for now.